pkgs/by-name/di/distant/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  perl,
+  pkg-config,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "distant";
+  version = "0.20.0";
+
+  src = fetchFromGitHub {
+    owner = "chipsenkbeil";
+    repo = pname;
```

`repo = pname` is an anti-pattern
```suggestion
    repo = "distant";
```